packages/run-protocol/test/bundleTool.js

```diff
@@ -90,7 +90,7 @@ export const makeBundleCache = (wr, cwd, readPowers, opts) => {
     return meta;
   };
 
-  const validate = async targetName => {
+  const validate = async (targetName, rootOpt) => {
```

Comment:
prefer that bundleTool changes get into master outside this PR. wouldn't be so much an issue if this will have a merge commit but looking at the commit history (e.g. "[chore(vault): fix ups after rebase](https://github.com/Agoric/agoric-sdk/pull/5129/commits/b586fe8c13d9d180ddcc436116f46dcabe6b351b)" i think a squash will be better. So request is either:
1. plan to squash, move out the bundleTool change, and update the PR title to be a conventional commit
2. plan to merge and clean up the commit list

Reply:
it should be a separate commit here directly from the source branch, so won't this get dropped out in rebase once that's in `master`?

Reply:
I don't follow. Are you planning to use automerge's `squash`, `merge` or `rebase`? 

Either way, I think what you want to do is before your next push fixup an earlier commit with https://github.com/Agoric/agoric-sdk/pull/5129/commits/b586fe8c13d9d180ddcc436116f46dcabe6b351b so it's not in the commit list.